.changeset/ten-fireants-march.md

```diff
@@ -0,0 +1,5 @@
+---
+'@backstage/plugin-jenkins': minor
+---
+
+Integrated the permission plugin, if enabled, the actions can be executed only by owned user (or assigned to related group)
```

Comment:
This isn't quite accurate - integrating with the permission policy puts control over who can perform actions in the hands of integrators. The specific users that will be permitted to perform the action will depend on what the integrator puts in their permission policy!